src/pages/CarDetails.jsx

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useEffect } from 'react';
+import { useNavigate, useParams, Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { useIsAuthenticated } from 'react-auth-kit';
+import { fetchSingleCar, deleteCar } from '../redux/cars/carsSlice';
+import formatTimestamp from '../components/Item';
+
+function CarDetails() {
```

Comment:
- [ ] I can add cars:

![image](https://github.com/AJ193/Final_Capstone_Front_End/assets/57421823/6e6658e2-e051-4d23-9c91-3200deff4859)

As you can see I've added some (the images are not showing up):

![image](https://github.com/AJ193/Final_Capstone_Front_End/assets/57421823/3a3475cc-94e1-491d-90d7-e8947b57c42d)

However, if I go to a car's details


![image](https://github.com/AJ193/Final_Capstone_Front_End/assets/57421823/100942d2-b7b1-4664-bfec-9508a0f3c035)

The car is not found:

![image](https://github.com/AJ193/Final_Capstone_Front_End/assets/57421823/d78640af-808b-4100-bfc8-f901ebd9e510)

It seems that the frontend is sending a request to the server for a "undefined id".

If I check the database, it seems to be working well, except for the fact that all the pictures are `nil`:

![image](https://github.com/AJ193/Final_Capstone_Front_End/assets/57421823/d13c4482-b5d5-4988-8cf7-5ee794642740)

## UPDATE I FOUND THE SOLUTION

I was able to correctly fetch the cars by reloading the page.

If I add a new car without reloading the page this happens:

![image](https://github.com/AJ193/Final_Capstone_Front_End/assets/57421823/f82b9044-232c-4947-ade1-60e25c51f2a7)

When I go to details:

![image](https://github.com/AJ193/Final_Capstone_Front_End/assets/57421823/1b1ba3c9-887c-4ced-9748-1fdd87dc3238)


If I reload the page:

![image](https://github.com/AJ193/Final_Capstone_Front_End/assets/57421823/0b6e286e-5ea2-452f-8848-c172c9e4280a)

If I go to details page:

![image](https://github.com/AJ193/Final_Capstone_Front_End/assets/57421823/fa669713-0ca2-4584-b896-24af08352763)

And this seems to be the problem:

![image](https://github.com/AJ193/Final_Capstone_Front_End/assets/57421823/21aa8a8c-f37c-4254-aa98-7ca178be02d2)

Every time a new car is added or deleted, a new request to the server needs to be made. 

Kindly, fix this bug so your app works as intended. 👍